packages/compiler/lib/service.ts

```diff
@@ -23,9 +23,11 @@ function getServiceDetails(program: Program) {
 export function setServiceNamespace(program: Program, namespace: NamespaceType): void {
   const serviceDetails = getServiceDetails(program);
   if (serviceDetails.namespace && serviceDetails.namespace !== namespace) {
+    /*
```

Comment:
Commented out diagnostics?

Reply:
I'm not entirely sure how this logic should work when there can be multiple service namespaces (due to one existing per-version). Let's discuss tomorrow.